commcare_connect/reports/tables.py

```diff
@@ -7,14 +8,28 @@
 from commcare_connect.opportunity.tables import SumColumn
 
 
+class AverageColumn(columns.Column):
```

Comment:
Are these two classes used?

Reply:
Yeah, i am going to remove these.

Reply:
Removed. https://github.com/dimagi/commcare-connect/pull/487/commits/50477d7a103ff72119d649207592ef275cee298a